mParticle-Apple-SDK/Next Gen/MPMediator.m

```diff
@@ -0,0 +1,23 @@
+//
+//  MPMediator.m
+//  mParticle-Apple-SDK
+//
+//  Created by Ben Baron on 6/29/23.
+//
+
+#import "MPMediator.h"
+
+@interface MPMediator()
+@property (nonatomic, strong) MPEventLogging *eventLogging;
+@end
+
+@implementation MPMediator
```

Comment:
Shouldn't kit be registered here and this serve as a glue for the components to interact and visit the kits?

Reply:
I had been thinking of the mediator as just the owner of the various components, but we could certainly move the loading of the mP Kit there if it makes sense. For this initial work, let's leave it as-is, but I'm about to start the spec to continue the work on the mP Kit, so we can include that there if it makes sense.